From: unknown author <unknown.author@example.com>
Date: Tue, 8 Feb 2011 15:10:31 +0100
Subject: [PATCH] python: fix long long autoconf test

New autoconf versions have AC_TYPE_LONG_LONG_INT, use it.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 configure.in |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 9b80a5d..2895a7d 100644
--- a/configure.in
+++ b/configure.in
@@ -1458,8 +1458,9 @@ AC_CHECK_SIZEOF(fpos_t, 4)
 AC_CHECK_SIZEOF(size_t, 4)
 AC_CHECK_SIZEOF(pid_t, 4)
 
+AC_TYPE_LONG_LONG_INT
 AC_CHECK_SIZEOF(long long)
-if test "$ac_cv_type_long_long" = "yes" ; then
+if test "$ac_cv_type_long_long_int" = "yes" ; then
   AC_DEFINE(HAVE_LONG_LONG, 1, [Define this if you have the type long long.])
 fi
 
